api/nakadi-event-bus-api.yaml

```diff
@@ -1246,7 +1254,16 @@ definitions:
     properties:
       name:
         type: string
-        description: Name of the strategy.
+        enum: ['hash', 'user_defined', 'random']
```

Comment:
I'd get rid of the description of the strategies in the API and the field as an enum. Otherwise it has to be kept consistent to the documentation of all existing strategies. Furthermore, the registered strategies are generally meant to be dynamic.